armi/utils/tests/test_hexagon.py

```diff
@@ -43,3 +44,43 @@ def test_numPositionsInRing(self):
         self.assertEqual(hexagon.numPositionsInRing(2), 6)
         self.assertEqual(hexagon.numPositionsInRing(3), 12)
         self.assertEqual(hexagon.numPositionsInRing(4), 18)
+
+    def test_rotatedCellCenter(self):
+        # Cell one is always the center of the hexagon and its position
+        # is rotation invariant
+        for rot in range(6):
+            self.assertTrue(hexagon.getIndexOfRotatedCell(1, rot), 1)
+
+    def test_rotatedFirstRing(self):
```

Comment:
@john-science here are two direct tests for `hexagon.getIndexOfRotatedCell` like you called out. I'm adding tests for the error conditions and changing the comment to a docstring on `test_rotatedCellCenter`. Is there anything you'd like to see beyond that?